pkg/crc/preflight/preflight_checks_tray_darwin.go

```diff
@@ -195,7 +195,9 @@ func downloadOrExtractTrayApp() error {
 	}()
 
 	logging.Debug("Trying to extract tray from crc executable")
-	err = embed.Extract(filepath.Base(constants.GetCRCMacTrayDownloadURL()), tmpArchivePath)
+	trayFileName := filepath.Base(constants.GetCRCMacTrayDownloadURL())
+	trayDestFileName := filepath.Join(tmpArchivePath, trayFileName)
```

Comment:
you could remove `archivePath` a bit further in this function. And that's quite some duplication between the windows and macos code :-/

Reply:
@cfergeau right now not want to refactor this part and remove duplication but will keep in mind.